packages/textarea/src/ux-textarea.ts

```diff
@@ -1,18 +1,21 @@
 import { customElement, bindable, inlineView } from 'aurelia-templating';
-import { DOM } from 'aurelia-pal';
+import { DOM, PLATFORM } from 'aurelia-pal';
 import { inject } from 'aurelia-dependency-injection';
 import { StyleEngine, UxComponent } from '@aurelia-ux/core';
 import { UxTextAreaTheme } from './ux-textarea-theme';
 import { observable } from 'aurelia-framework';
-import * as VIEW from './ux-textarea.html';
+import VIEW from './ux-textarea.html';
 
 export interface UxTextAreaElement extends HTMLElement {
   value: string;
 }
 
 @inject(Element, StyleEngine)
 @customElement('ux-textarea')
-@inlineView(VIEW)
+@inlineView(
+  VIEW,
+  [PLATFORM.moduleName('@aurelia-ux/textarea/ux-textarea.css')]
```

Comment:
Using inlineView with css dep is fine for cli bundler, even when without `PLATFORM.moduleName` wrapper.

However, **this does not work for webpack**. This is a known issue, because this css dep is introduced from a js file, our default webpack config will use `style-loader` to inject it to HTML head (instead of leaving the responsibility to aurelia).

There are two issues in webpack.
1. doesn't support 2nd param in inlineView https://github.com/aurelia/webpack-plugin/issues/140
2. inline css dep through decorator doesn't work https://github.com/aurelia/cli/issues/911

The only solution is to avoid using inlineView, always use conventional html pair. You can only use inlineView if it doesn't introduce any css dep.

Reply:
It doesn't matter the css dep is inside the template string of inlineView, or the 2nd param of inlineView. The webpack config see them same as the css dep is introduced from a js file.

Reply:
Thanks for the input here, I forgot about the behavior of style loader and the like